frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/TopBar.js

```
@@ -1,156 +1,308 @@
+      <div> <img
+        src="https://tse4.mm.bing.net/th/id/OIP.LcAu4hLmyz-LQqUVPtVC9AHaFj?r=0&rs=1&pid=ImgDetMain&o=7&rm=3"
+        alt="Logo"
+        style={{
+          width: "59px",
+          height: "51px",
+          marginRight: "16px",
+          borderRadius: "50%",
+        }} /></div>
+      {/* Right Side */}
+      <div style={{ display: "flex", alignItems: "center", gap: "20px", marginLeft: "auto" }}>
+        <button onClick={() => document.documentElement.style.fontSize = "smaller"} style={{ background: "none", border: "none" }}>A-</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)